src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -0,0 +1,21 @@
+(ns clojure-tictactoe.game.game-rules
+  (:require [clojure-tictactoe.game.board :as board]))
+
+
+(defn assess-win 
+  [board]
+  (cond 
+    (= (board 0) (board 1) (board 2)) (board 0)
+    (= (board 3) (board 4) (board 5)) (board 3)
+    (= (board 6) (board 7) (board 8)) (board 6)
+    (= (board 0) (board 3) (board 6)) (board 0)
+    (= (board 1) (board 4) (board 7)) (board 1)
+    (= (board 2) (board 5) (board 8)) (board 2)
+    (= (board 0) (board 4) (board 8)) (board 0)
+    (= (board 2) (board 4) (board 6)) (board 2)))
+
+(defn game-over?
+  [board]
+  (not (not (or 
```

Comment:
Do you need a double `not` here?

Reply:
You need it because of the way that the code's currently set up, yes. But refactoring so that `or` only had to determine whether the values given to it are true or false would help simplify this. I.E. - Don't give `or` the option to return anything else but a boolean. Which you can actually do here.